draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -9645,6 +9656,9 @@ Content-Type: text/plain
     &Note; Some recipients attempt to recover from Location fields
     that are not valid URI references. This specification does not mandate or
     define such processing, but does allow it for the sake of robustness.
+    A Location field value that has multiple comma-separated members
+    poses difficulties for recovery due to the value's syntax; as a result,
+    recovery in that situation is not interoperable across implementations.
```

Comment:
```suggestion
    A Location field value cannot allow a list of members because the comma list separator
    is a valid data character within a URI-reference. If an invalid message is sent with multiple
    Location field instances, a recipient along the path might combine the field instances into
    one value. Recovery of a valid Location field value from that situation is difficult and not
    interoperable across implementations.
```